reporters/dogstatsd.go

```diff
@@ -9,7 +9,10 @@ package reporters
 
 import (
 	"fmt"
+	"sync"
+	"time"
 
+	"github.com/DataDog/datadog-go/statsd"
```

Comment:
We already have the dogstatsd down below. Is it necessary to have both?

Reply:
I need to import this package to be able to access the `statsd.Client` which has the `Close` function, as I mentioned in [this comment](https://github.com/topfreegames/maestro/pull/545#discussion_r1044629580).